nncf/pruning/filter_pruning/algo.py

```diff
@@ -472,10 +485,14 @@ def _set_binary_masks_for_pruned_modules_globally_by_flops_target(self,
             cumulative_filters_importance = torch.zeros(filters_num[0]).to(device)
             # Calculate cumulative importance for all filters in this group
             for minfo in cluster.nodes:
-                normalized_weight = self.weights_normalizer(minfo.module.weight)
-                filters_importance = self.filter_importance(normalized_weight,
+                # TODO: check why here normalization is needed (in LEGR case normalization shouldn't be done)
+                # normalized_weight = self.weights_normalizer(minfo.module.weight)
+                weight = minfo.module.weight
+                filters_importance = self.filter_importance(weight,
                                                             minfo.module.target_weight_dim_for_compression)
-                cumulative_filters_importance += filters_importance
+                scaled_importance = self.ranking_coeffs[minfo.module_scope][0] * filters_importance + \
+                                    self.ranking_coeffs[minfo.module_scope][1]
+                cumulative_filters_importance += scaled_importance
```

Comment:
Is there a way to dump the learned layer-wise ranking coefficients with the model checkpoint, for instance?

Reply:
Of course, saving to the checkpoint is reasonable.
I have also idea to save ranking coefficients to separate file to allow easy saving/loading to fine-tune with different PR.
Which option do you prefer?

Reply:
@vanyalzr As discussed, in the current implementation coefficients aren't useful in the checkpoint (to re-prune the model with another PR). 
Therefore, as a temporary solution, the coefficients will be saved to a separate file as it is convenient to use. 
When the mask applying scheme in pruning will be fixed (ticket 53524) I will move coefficients to the model and/or checkpoint.